src/utils/hact/workOrderStatusUpdate/updateWorkOrder.js

```diff
@@ -3,7 +3,8 @@ export const buildWorkOrderUpdate = (
   addedTasks,
   reference,
   variationReason,
-  isOvertime
+  isOvertime,
+  isOvertimeUpdate
```

Comment:
Do we need this additional flag? We seem to always set it to `true` when `isOvertime` is supplied. Therefore can't we remove it and just check for the presence of `isOvertime` when building the JSON as we had before?